fzf-maccy.sh

```diff
@@ -3,9 +3,9 @@
 set -eu -o pipefail
 
 DB=~/Library/Containers/org.p0deje.Maccy/Data/Library/Application\ Support/Maccy/Storage.sqlite
-SQL="select   distinct ZVALUE
+SQL="select   distinct TRIM(ZVALUE)
```

Comment:
I prefer to keep the leading whitespaces, especially when copying code snippets.

Reply:
I've pushed a new commit which doesn't use TRIM() (see below).